### DISCUSSION TITLE
HTTP Error 409

### DISCUSSION THREAD

**Discussion by IndrajeethY** [deta/cloud-docs#335] [Help] [answered]
hey i tryied to use deta-python  i tried to insert same key two times. I've got this exception
`packages/deta/base.py", line 124, in insert
    code, res = self._request("/items", "POST", {"item": data}, content_type=JSON_MIME)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/deta/service.py", line 85, in _request
    raise urllib.error.HTTPError(url, status, res.reason, res.headers, res.fp)
urllib.error.HTTPError: HTTP Error 409: Conflict`
is there any way to except this exception on deta-python natively? or cant just deta replace new key with old key if two is of same name?

**Comment by jnsougata** [comment, marked as answer]
the method `insert` only adds a new field inside the base if the key doesn’t exist inside the base previously. if the key already exists it will raise the error which one you are seeing right now. you can send updates to the pre existing key or just overwrite the field of the pre existing key using `put` method

**Reply by IndrajeethY** [reply]
okie tqs
